File: src/main/java/no/nav/service/pensjon/regler/to/BeregnPoengtallBatchResponse.java
package no.nav.service.pensjon.regler.to;

import java.util.Vector;

import no.nav.domain.pensjon.regler.grunnlag.PersonOpptjeningsgrunnlag;

public class BeregnPoengtallBatchResponse extends ServiceResponse{
	private Vector<PersonOpptjeningsgrunnlag> personOpptjeningsgrunnlagListe = new Vector<PersonOpptjeningsgrunnlag>();

}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnPoengtallBatchResponse.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.grunnlag.PersonOpptjeningsgrunnlag


class BeregnPoengtallBatchResponse : ServiceResponse() {
    var personOpptjeningsgrunnlagListe: MutableList<PersonOpptjeningsgrunnlag> = mutableListOf()
}